server/app.js

```diff
@@ -26,17 +27,21 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', index);
 app.use('/users', users);
+ serverSetUpAdminDummyPage
```

Comment:
this is not resolved properly - you should delete these lines serverSetup.... and master. This will break the code.